### DISCUSSION TITLE
An error occurred while generating your metrics :( Internal Server Error: failed to process metrics correctly

### DISCUSSION THREAD

**Discussion by 0xvNFT** [lowlighter/metrics#1585] [General]
An error occurred while generating your metrics :(
Internal Server Error: failed to process metrics correctly

**Comment by Pyenb** [comment]
Are you using the github action? Were there any other errors? Did you try rerunning the action?

**Reply by 0xvNFT** [reply]
Tested it on website and facing issues.

**Reply by Pyenb** [reply]
Try using the GitHub action instead of the Website. Gives you more  flexibility and less dependence on the server 👍

Read [the docs](https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/documentation/setup/action.md) for more info.

**Comment by anisurrahman75** [comment]
I am also facing this error when I click on the 'Generate your metrics!' button.

**Reply by Pyenb** [reply]
Try using the GitHub action instead of the Website 👍

Read the docs for more info.

**Reply by anisurrahman75** [reply]
Yes, it's worked, Thank's
